### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by ghost**
## Summary

- Navbar

## Screenshots

Captura de pantalla de 2019-05-23 12-37-54

## Trello Card

https://trello.com/c/zs176bSu/13-navbar

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div class="navbar">
```
Comment (Dorian30) [inline maquetado/index.html]: You can use `<nav>... </nav>` tag for this and keep the same class name

File: maquetado/index.html [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+    <img class="navbar-logo" src="./assets/wolox_logo.svg">
```
Comment (Dorian30) [inline maquetado/index.html]: Remember to add the `alt` attribute to your images. Good work with the naming here.

File: maquetado/scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
+.navbar {
+  align-items: center;
+  background: linear-gradient(to bottom, #bbbaba, #c2bfbf, #d4d2d2, #f7f5f5) bottom no-repeat;
```
Comment (Dorian30) [inline maquetado/scss/navbar.scss]: Interesting approach, but you can use `box-shadow` to make it simpler :+1:

File: maquetado/scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  background-color: $white;
+  background-size: 100% 10px;
```
Comment (Dorian30) [inline maquetado/scss/navbar.scss]: Same, with `box-shadow`  you won't have to do this.

File: maquetado/scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+}
+
+.navbar-logo {
+  height: $nav-logo-height;
+  margin-left: 20px ;
```
Comment (Dorian30) [inline maquetado/scss/navbar.scss]: You have a white space here after the semicolon.

```suggestion
margin-left: 20px;
```

**Review by Dorian30 — CHANGES_REQUESTED** [review]

File: maquetado/index.html [hunk 6]
```diff
@@ -49,25 +53,25 @@ <h2 class="title">Back-End Tools</h2>
...
       <div class="technology">
-        <img class="logo" src="./assets/aspnet_icon.png">
+        <img class="logo" src="./assets/aspnet_icon.png" alt="technology logo">
         <h4 class="technology-title">ASP.NET Core</h4>
         <h4 class="technology-release">Release:2016</h4>
         <p class="technology-description">ASP.NET Core is the next generation of ASP.NET. It is a modular framework that runs on both the full .NET Framework, on Windows, and the cross-platform .NET Core.</p>
       </div>
       <div class="technology">
-        <img class="logo" src="./assets/django_icon.svg">
+        <img class="logo" src="./assets/django_icon.svg" alt="technology logo">
```
Comment (Dorian30) [inline maquetado/index.html]: Same as the content layout PR. Alt attribute is used to give some context about what the image is about, but let's do this fix in content layout PR.

File: maquetado/scss/navbar.scss (hunk 4 above)
Comment (Dorian30) [reply maquetado/scss/navbar.scss]: Remember to delete this line. It's no needed.

**Review by Dorian30 — APPROVED** [review]

File: maquetado/index.html [hunk 7]
```diff
@@ -10,35 +10,39 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
+  <nav class="navbar">
+    <img class="navbar-logo" src="./assets/wolox_logo.svg" alt="wolox logo">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: Close this image tag `/>`

File: maquetado/index.html [hunk 8]
```diff
@@ -10,35 +10,39 @@
...
+    <h2 class="navbar-text">Developer Tools</h2>
+  </nav>
   <h2 class="title">Front-End Tools</h2>
   <div class="container">
     <div class="wrap">
       <div class="technology">
-        <img class="logo" src="./assets/react_icon.svg">
+        <img class="logo" src="./assets/react_icon.svg" alt="technology logo">
```
Comment (mmamaniwolox) [inline maquetado/index.html]: This should be a descriptive alt, for example, `alt="React logo"`

File: maquetado/scss/navbar.scss [hunk 9]
```diff
@@ -0,0 +1,21 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0px 10px 13px -7px #000000, 0px 7px 5px 0px rgba(125,125,125,0.43);
```
Comment (mmamaniwolox) [inline maquetado/scss/navbar.scss]: Use color constants for `#0000` and `rgba(....)`

I also dont think you need so many parameters here, the 3 first and a color should be enought

File: maquetado/scss/navbar.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+  height: $nav-height;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+}
+
+.navbar-logo {
+  height: $nav-logo-height;
+  margin-left: 20px;
```
Comment (mmamaniwolox) [inline maquetado/scss/navbar.scss]: I would set `margin` here, you have separated it from the left border, but what if you need to add another element beside this logo?

File: maquetado/scss/navbar.scss [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.navbar-text {
+  font-size: $text-big;
+  margin-right: 20px;
```
Comment (mmamaniwolox) [inline maquetado/scss/navbar.scss]: Same, try not to be so specific and more generic in this cases

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/scss/fonts.scss [hunk 12]
```diff
@@ -1,3 +1,5 @@
+$nav-height: 90px;
```
Comment (mmamaniwolox) [inline maquetado/scss/fonts.scss]: This are not fonts, you should move them to `navbar.css`

Same with `title-decoration`, should be in `titles.scss`

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**